src/main/java/org/gridsuite/network/map/NetworkMapService.java

```diff
@@ -1460,6 +1479,23 @@ public List<String> getHvdcLinesIds(UUID networkUuid, String variantId, List<Str
         }
     }
 
+    public Object getBranchOrThreeWindingsTransformer(UUID networkUuid, String variantId, String equipmentId) {
```

Comment:
maybe add a comment to explain why we try network.getLine and then getTwoWindingsTransformer and then getThreeWindingsTransformer instead of using getIdentifiable, or instead of sending the type of equipment in the request

Reply:
Ok done